site/themes/arangodb-docs-theme/static/css/theme.css

```
@@ -592,6 +592,92 @@ a.section-link {
+    color: var(--gray-700);
+    border: 2px solid var(--gray-300);
+    font-family: var(--font-headlines);
+    font-weight: 700;
+    font-size: 0.95rem;
+    line-height: 1;
+}
+
+.steps .step-icon svg {
+    width: 16px;
+    height: 16px;
+    fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)